bootstrap.py

```diff
@@ -339,7 +338,9 @@ def install_puppet_agent():
     print_generic("Visit the UI and approve this certificate via Infrastructure->Capsules")
     print_generic("if auto-signing is disabled")
     exec_failexit("/usr/bin/puppet agent --test --noop --tags no_such_tag --waitforcert 10")
-    exec_failexit("/sbin/service puppet restart")
+    if 'puppet-enable' not in options.skip:
+        exec_failexit("/sbin/service puppet restart")
```

Comment:
I'd have sorted those two the other way round, first enable, then restart, but it does not strictly hurt.